nuxeo/operations.py

```diff
@@ -221,33 +221,46 @@ def get_attributes(operation, **kwargs):
         if operation:
             command = operation.command
             input_obj = operation.input_obj
+            context = operation.context
             params = operation.params
         else:
             command = kwargs.pop('command', None)
             input_obj = kwargs.pop('input_obj', None)
+            context = kwargs.pop('context', None)
             params = kwargs.pop('params', kwargs)
-        return command, input_obj, params
+        return command, input_obj, params, context
+
+    def build_payload(self, params, context):
```

Comment:
Could you add a docstring and types comment please?

Reply:
@BoboTiG Just added both to the PR.